src/sentry/incidents/grouptype.py

```diff
@@ -13,13 +13,21 @@
 from sentry.ratelimits.sliding_windows import Quota
 from sentry.types.group import PriorityLevel
 from sentry.workflow_engine.handlers.detector import DetectorOccurrence, StatefulDetectorHandler
+from sentry.workflow_engine.handlers.detector.base import EvidenceData
 from sentry.workflow_engine.models.data_source import DataPacket
 from sentry.workflow_engine.types import DetectorGroupKey, DetectorSettings
 
 COMPARISON_DELTA_CHOICES: list[None | int] = [choice.value for choice in ComparisonDeltaChoices]
 COMPARISON_DELTA_CHOICES.append(None)
 
 
+@dataclass
+class MetricIssueEvidenceData(EvidenceData):
+    detector_id: int
+    alert_id: int
+    data_condition_ids: list[int]
```

Comment:
🤔 can we have the `detector_id` and `data_condition_ids` on the EvidenceData class? (my thought is anything related to workflow engine that can be done in the abstraction can also be hidden away in the platform)